src/components/02_atoms/Widgets/DatetimeTimestamp.js

```diff
@@ -16,9 +16,9 @@ const DatetimeTimestamp = ({
     fullWidth
     label={label}
     type="datetime-local"
-    defaultValue={value}
+    defaultValue={new Date(value * 1000).toJSON().slice(0, 19)}
```

Comment:
May I ask a question: Why do we not provide a default value from the schema?

Reply:
The value is coming from state. JavaScipt uses millisecond timestamp rather than second based used more commonly (which we also use for state) meaning that we have to do some conversions when using JavaScript APIs.